doc/neps/nep-0036-fair-play.rst

```diff
@@ -0,0 +1,177 @@
+==================
+NEP 36 — Fair Play
+==================
+
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2019-10-24
+:Resolution: Draft
+
+
+Abstract
+--------
+
+This document describes Rules of Play for engaging with the NumPy
+project as an external entity.  It discusses usage of the name NumPy,
+modified distributions of NumPy, and how to best communicate
+with the project when patched versions are deployed.
+
+Companies and developers will know after reading this NEP what kinds
+of behavior the community would like to see, and which we consider
+troublesome, bothersome, and unacceptable.
+
+Motivation
+----------
+
+It has come to our attention that modified versions of NumPy are
+distributed by external entities.  Sometimes, the patches are
+innocuous, such as when they allow NumPy to function on the target
+system.  In other cases, they add new functionality which causes
+problems for the community.
+
+For example, in December 2018, through the filing of `Issue #12515
+<https://github.com/numpy/numpy/issues/12515>`__, it became known to
+the NumPy project that an external company had published a version of
+NumPy with an extended footprint.  In this specific instance, a bug
+was filed against the `np.erf` function---a function that NumPy itself
+had never provided.
+
+In the months that followed, several conversations was had with the
+entity involved, and a better understanding emerged of needs on either
+side.  The entity agreed to publish the patches made against Python,
```

Comment:
made against python or against numpy? Do they still ship the modified version?

Reply:
`np.erf` and `np.invsqrt` were removed. They still remain in `np.core.umath` in the patched version. 

The plan is to remove extensive patching of NumPy and instead build a scikit-* package which exposes MKL/SVML-backed universal functions.